dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
-black==21.8b0
-mypy==0.910
-pylint==2.6.0
-pytest==6.2.2
-pytest-cov==2.11.1
-xenon==0.7.3
+black >= 21.8b0
```

Comment:
do we also want to do a `less than` the next major version for these? This feels a bit risky

Reply:
`>=` is a pretty standard way to do it for libraries. Some examples:

[pre-commit](https://github.com/pre-commit/pre-commit/blob/main/setup.cfg)
[flask](https://github.com/pallets/flask/blob/main/setup.py)
[mypy](https://github.com/python/mypy/blob/master/mypy-requirements.txt)